pulse/service/src/main/java/gov/ca/emsa/pulse/service/PatientSearchTermsException.java

```diff
@@ -3,5 +3,5 @@
 import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.ResponseStatus;
 
-@ResponseStatus(value = HttpStatus.BAD_REQUEST, reason = "One of the following fields were blank: DOB, Family/Given Name, Gender")
+@ResponseStatus(value = HttpStatus.BAD_REQUEST, reason = "One of the following fields were blank or incorrect: DOB, Family/Given Name, Gender")
```

Comment:
This is a TOTAL aside but it would be cool to localize the application so if we ever wanted to run it in another country where they do prefer "given" and "family" names vs here where we prefer "first" and "last" the strings could be swapped out.